docs/docs-content/integrations/community_packs.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_label: "Community Packs"
+title: "Community Packs"
+description: "Identify Community Packs in Palette, which are not actively maintenance."
```

Comment:
please check this description, it is not worded properly

Reply:
Thanks for catching this!